Makefile

```diff
@@ -23,6 +23,14 @@ marimo/_lsp: $(shell find lsp)
 py:
 	pip install -e ".[dev]"
 
+.PHONY: dev
+dev:
```

Comment:
Maybe better suited as a task?

Reply:
True!

Reply:
i think its fine as `make` task actually, so we have 1 task running (`make`) and people can use more bare-metal CLI's if needed